src/main/java/org/mifos/processor/bulk/API/implementation/BatchTransactionsController.java

```diff
@@ -0,0 +1,74 @@
+package org.mifos.processor.bulk.api.implementation;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.camel.Exchange;
+import org.apache.camel.ProducerTemplate;
+import org.json.JSONObject;
+import org.mifos.processor.bulk.api.definition.BatchTransactions;
+import org.mifos.processor.bulk.file.FileStorageService;
+import org.mifos.processor.bulk.utility.Headers;
+import org.mifos.processor.bulk.utility.SpringWrapperUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.multipart.MultipartException;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+import static org.mifos.processor.bulk.zeebe.ZeebeVariables.*;
+@Slf4j
+@RestController
+public class BatchTransactionsController implements BatchTransactions {
+
+    @Autowired
+    private ProducerTemplate producerTemplate;
+
+    @Autowired
+    ObjectMapper objectMapper;
+
+    @Autowired
+    FileStorageService fileStorageService;
+
+    @SneakyThrows
+    @Override
+    public String batchTransactions(HttpServletResponse httpServletResponse,
+                                    String requestId, MultipartFile file, String fileName,
+                                    String purpose, String type, String tenant) throws IOException {
+        assert (file.getSize() >0);
```

Comment:
what is assert doing here? Please handle validations and exceptions in standard way

Reply:
removed